R/autoplot.curve_params.R

```diff
@@ -2,27 +2,37 @@
 #'
 #' @inheritParams plot_curve_params_one_ab
 #' @inheritDotParams plot_curve_params_one_ab
-#' @param antigen_isos antigen isotypes to analyze (can be used to subset `curve_params`)
+#' @param antigen_isos antigen isotypes to analyze (can subset `curve_params`)
 #' @param ncol how many columns of subfigures to use in panel plot
 #' @details
 #' ## `rows_to_graph`
-#' Note that if you directly specify `rows_to_graph` when calling this function, the row numbers are enumerated separately for each antigen isotype; in other words, for the purposes of this argument, row numbers start over at 1 for each antigen isotype. There is currently no way to specify different row numbers for different antigen isotypes; if you want to do that, you will could call [plot_curve_params_one_ab()] directly for each antigen isotype and combine the resulting panels yourself. Or you could subset `curve_params` manually, before passing it to this function, and set the `n_curves` argument to `Inf`.
+#' Note that if you directly specify `rows_to_graph` when calling this function,
+#' the row numbers are enumerated separately for each antigen isotype;
+#' in other words, for the purposes of this argument,
+#' row numbers start over at 1 for each antigen isotype. There is currently
```

Comment:
In general, please line-break after every sentence; it makes code review easier later on.

Reply:
@d-morrison question about this... in this case the sentences were too long to pass through linting (>80 characters), which is why I had to break them at weird points. What do you suggest for the future?

Reply:
@kristinawlai ideally, add line breaks at all sentence ends _and also_ at additional points to make all lines less than 80 characters; good places to break are after commas and before/after grammatical substructures (noun-phrases, verb-phrases, etc). I've recently started experimenting with turning my prose source code into an almost poetry-style format; the compiler will ignore single line breaks anyway and convert it back into a "normal" format, so you can have some fun with the source code formatting.